docs/src/content/docs/guides/pages.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Pages
+description: Learn how to create and manage your documentation site’s pages with Starlight.
+sidebar:
+  order: 1
+---
+
+Starlight generates your site’s HTML pages based on your content, with flexible options provided via Markdown frontmatter.
+In addition, Starlight projects have full access to Astro’s powerful page generation tools.
+This guide shows how page generation works in Starlight.
+
+## Content pages
+
+### File formats
+
+Starlight supports authoring content in Markdown and MDX with no configuration required.
+You can add support for Markdoc by installing the experimental [Astro Markdoc integration](https://docs.astro.build/en/guides/integrations-guide/markdoc/).
+
+### Add pages
+
+Add new pages to your site by creating `.md` or `.mdx` files in `src/content/docs/`.
+Use sub-folders to organize your files and to create multiple path segments.
+
+For example, the following file structure will generate pages at `example.com/hello-world` and `example.com/reference/faq`:
+
+import FileTree from '~/components/file-tree.astro';
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - hello-world.md
+      - reference/
+        - faq.md
+
+</FileTree>
+
+### Type-safe frontmatter
+
+All Starlight pages share a customizable [common set of frontmatter properties](/reference/frontmatter/) to control how the page appears:
+
+```md
+---
+title: Hello, World!
+description: This is a page in my Starlight-powered site
+---
+```
+
+If you forget anything important, Starlight will let you know.
+
+## Custom pages
+
+For advanced use cases, you can create custom pages in Astro’s `src/pages/` directory.
+This is helpful if you need to build pages with a completely custom layout or generate a page from an alternative data source.
+The `src/pages/` directory uses file-based routing and includes support for `.astro` files.
```

Comment:
Just checking: ONLY `.astro` pages? Not HTML, MDX etc. Because that's what this implies.

Reply:
No, all formats Astro supports are supported, this is what the “includes” was intended to imply. I didn’t want to duplicate the full list from Astro docs when people should use that as the source of truth.